src/components/table/Badges.tsx

```diff
@@ -2,27 +2,20 @@ import { DiscriminatedItem, PackedItem } from '@graasp/sdk';
 import { ItemBadges } from '@graasp/ui';
 
 import { useBuilderTranslation } from '../../config/i18n';
-import { hooks } from '../../config/queryClient';
 import { BUILDER } from '../../langs/constants';
 
-const { useManyItemPublishedInformations } = hooks;
-
 type ItemStatuses = {
   showChatbox: boolean;
   isPinned: boolean;
   isCollapsible: boolean;
   isHidden: boolean;
-  isPublic: boolean;
```

Comment:
Public is not the same as Published. Why are you removing this?

Reply:
Did not know that, will re-add this.